Not compiling on windows

Hi, I get the following errors when a crate I'm using trys to compile this one:

```shell
error[E0425]: cannot find value `revision` in this scope
   --> C:\Users\dkolsoi\.cargo\registry\src\github.com-1ecc6299db9ec823\headless_chrome-0.1.4\src\browser\fetcher.rs:250:12
    |
250 |         if revision.as_ref().parse::<u32>()? > 591_479 {
    |            ^^^^^^^^ help: a local variable with a similar name exists: `_revision`

error[E0425]: cannot find function `get_chrome_path_from_registry` in this scope
   --> C:\Users\dkolsoi\.cargo\registry\src\github.com-1ecc6299db9ec823\headless_chrome-0.1.4\src\browser\mod.rs:367:29
    |
367 |         if let Some(path) = get_chrome_path_from_registry() {
    |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
    |
1   | use crate::browser::process::get_chrome_path_from_registry;
    |
```

It seems as though `_revision` is the actual variable name in the former and `get_chrome_path_from_registry` isn't imported in the latter

--- a/src/browser/fetcher.rs
+++ b/src/browser/fetcher.rs
@@ -247,7 +247,7 @@ fn archive_name<R: AsRef<str>>(_revision: R) -> Result<&'static str, Error> {
     #[cfg(windows)]
     {
         // Windows archive name changed at r591479.
-        if revision.as_ref().parse::<u32>()? > 591_479 {
+        if _revision.as_ref().parse::<u32>()? > 591_479 {
             Ok("chrome-win")
         } else {
             Ok("chrome-win32")

--- a/src/browser/fetcher.rs
+++ b/src/browser/fetcher.rs
@@ -233,21 +233,25 @@ where
     }
 }
 
-fn archive_name<R: AsRef<str>>(_revision: R) -> Result<&'static str, Error> {
+fn archive_name<R: AsRef<str>>(revision: R) -> Result<&'static str, Error> {
     #[cfg(target_os = "linux")]
     {
+        drop(revision);
+
         Ok("chrome-linux")
     }
 
     #[cfg(target_os = "macos")]
     {
+        drop(revision);
+
         Ok("chrome-mac")
     }
 
     #[cfg(windows)]
     {
         // Windows archive name changed at r591479.
-        if _revision.as_ref().parse::<u32>()? > 591_479 {
+        if revision.as_ref().parse::<u32>()? > 591_479 {
             Ok("chrome-win")
         } else {
             Ok("chrome-win32")

--- a/src/browser/mod.rs
+++ b/src/browser/mod.rs
@@ -364,6 +364,8 @@ pub fn default_executable() -> Result<std::path::PathBuf, String> {
 
     #[cfg(windows)]
     {
+        use crate::browser::process::get_chrome_path_from_registry;
+
         if let Some(path) = get_chrome_path_from_registry() {
             if path.exists() {
                 return Ok(path);

--- a/src/browser/process.rs
+++ b/src/browser/process.rs
@@ -39,7 +39,7 @@ enum ChromeLaunchError {
 }
 
 #[cfg(windows)]
-fn get_chrome_path_from_registry() -> Option<std::path::PathBuf> {
+pub(crate) fn get_chrome_path_from_registry() -> Option<std::path::PathBuf> {
     RegKey::predef(HKEY_LOCAL_MACHINE)
         .open_subkey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe")
         .and_then(|key| key.get_value::<String, _>(""))

